ethcore/executive-state/src/lib.rs

```diff
@@ -1542,7 +1542,7 @@ mod tests {
 	fn should_not_panic_on_state_diff_with_storage() {
 		let mut state = get_temp_state();
 		let a = Address::from_low_u64_be(0xa);
-		state.init_code(&a, b"abcdefg".to_vec()).unwrap();
+		state.init_code(&a, b"abcdefg".to_vec()).unwrap();;
```

Comment:
`;;`? :smile:

Reply:
I've added a ton of these in these PRs, I have no idea how it happened. :/